test/connection.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env node, mocha */
+require('./init.js');
+var assert = require('assert');
+var DataSource = require('loopback-datasource-juggler').DataSource;
+
+var config;
+
+before(function() {
+  config = global.getConfig();
+  console.log("config", config);
+});
+
+describe('testConnection', function() {
+  it('should pass with valid settings', function(done) {
+    var db = new DataSource(require('../'), config);
```

Comment:
Please save `require('../')` into a variable, e.g. `connector` or `mysqlConnector` and use the variable everywhere in the new tests.